kotlin-semver/src/jsMain/generated/semver/functions/sort.kt

```diff
@@ -0,0 +1,28 @@
+// Generated by Karakum - do not modify it manually!
+
+@file:JsModule("semver/functions/sort")
+
+package semver.functions
+
+import semver.classes.SemVer
+
+/* import SemVer = require("../classes/semver"); */
+
+/* import semver = require("../index"); */
+
+/**
+ * Sorts an array of semver entries in ascending order using `compareBuild()`.
+ */
+external fun <T : Any /* string | SemVer */> sort(list: Array<T>): Array<T>
```

Comment:
Could you please report bug on Karakum with cases where `Array` generated?
In all of them `ReadonlyArray` expected :(

Reply:
Sure, how about `Error`?
Those are not here but when I prepare PR for `null-writable` where I used a plugin to change them to `js.errors.JsError`.

Reply:
I created a PR for Karakum at https://github.com/karakum-team/karakum/pull/28 and for now added a plugin here that changes the types.